.github/workflows/auto_tag_and_release.yml

```diff
@@ -0,0 +1,37 @@
+name: Auto tag version bump commit and create release
+
+on:
+  push:
+    branches: [ master]
+    tags-ignore:
+      - '**'
+
+jobs:
+  tag_version_bump_commit:
+    if: startsWith(github.event.head_commit.message, '[Version Bump]')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: "Configure Git Credentials"
+        run: |
+          git config user.name "${{ github.actor }}"
+          git config user.email "${{ github.actor }}@users.noreply.github.com"
+
+      - name: "Create and push git tag"
+        id: version
+        run: |
+          VERSION="$(grep -o -e RELEASE_VERSION\=\"[0-9]*\.[0-9]*\.[0-9]*  dotnet/build-packages.sh | sed 's/RELEASE_VERSION="//')"
+          TAGNAME=dotnet-v$VERSION
+          git tag "$TAGNAME"
+          git push origin "$TAGNAME"
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
+        with:
+          draft: true
+          name: "$TAGNAME"
+          tag_name: "$TAGNAME"
+          prerelease: false
+          body: "copy paste from the previous release"
```

Comment:
> copy paste from the previous release

If it's going to be static text, then may as well include it here, as we can fix links too? (Below assumes that tracer version and aas version are the same). We should also probably parse the datadog agent version in the previous step, to ensure that stays up to date
```suggestion
          body: |
          
          ## Versions included
          [.NET Tracer v${{steps.version.outputs.version}}](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v${{steps.version.outputs.version}})
          [Datadog Agent v7.32.4](https://github.com/DataDog/datadog-agent/releases/tag/7.32.4)
          
          Hosted on [Nuget](https://www.nuget.org/packages/Datadog.AzureAppServices.DotNet/${{steps.version.outputs.version}})
          
          [Site Extension Documentation](https://docs.datadoghq.com/serverless/azure_app_services/)
          
          ## Requirements
          The .NET Datadog APM Site Extension requires that you setup the [Microsoft Azure Integration](https://docs.datadoghq.com/integrations/azure_app_services/) first.
          Please follow the directions in the referenced document.
          
          ## .NET Core Installation and Upgrade
          Install or upgrade the site extension, and verify traces and metrics.
          
          ## .NET Framework Installation and Upgrade
          Fully stop your web app before installing, modifying, or removing the .NET Datadog APM Site Extension.
          This site extension uses the [.NET Profiling API](https://docs.microsoft.com/en-us/dotnet/framework/unmanaged-api/profiling/profiling-interfaces) which hooks in at process start.
          *Restart* recycles an application pool, the app must be *STOPPED* before any changes to this extension.

```